packages/web-client/tests/acceptance/issue-prepaid-card-persistence-test.ts

```diff
@@ -75,14 +75,15 @@ module('Acceptance | issue prepaid card persistence', function (hooks) {
       'service:workflow-persistence'
     );
 
+    // eslint-disable-next-line ember/no-array-prototype-extensions
```

Comment:
This made me investigate what the heck this rule is doing that would cause it to misfire here, and the whole rule is extremely dubious. We can turn it off globally.

Reply:
https://github.com/ember-cli/eslint-plugin-ember/pull/1562